src/sentry/tempest/endpoints/tempest_credentials.py

```diff
@@ -50,7 +51,9 @@ def post(self, request: Request, project: Project) -> Response:
         serializer = DRFTempestCredentialsSerializer(data=request.data)
         serializer.is_valid(raise_exception=True)
         try:
-            serializer.save(created_by_id=request.user.id, project=project)
+            credentials = serializer.save(created_by_id=request.user.id, project=project)
+            # Make initial call to determine the latest item ID
+            fetch_latest_item_id.delay(credentials.id)
```

Comment:
how exepensive is fetching of the latest item? It might happen that we manually call this task, and in the next few milliseconds periodic task collects this credentials, see that there is no last item and calls this task again.

Will we have problem if that happens? Or is it ok if we fetch that twice in a short period of time?